README.md

```diff
@@ -28,69 +25,75 @@
 
-The ngrok Ingress Controller for Kubernetes is an open source controller for adding public and secure ingress traffic to your K8s services. If you’ve used ngrok in the past, you can think of the ingress controller as the ngrok agent built as an idiomatic K8s resource — available as a helm chart, configurable via K8s manifests, scalable for production usage, and leveraging Kubernetes best practices.
+Leverage [ngrok](https://ngrok.com/) for your [Kubernetes Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/). Instantly add load balancing, authentication, and observability to your services via ngrok Cloud Edge modules using Custom Resource Definitions (CRDs) and Kubernetes-native tooling.
 
-The ngrok Ingress Controller for Kubernetes lets developers define public and secure ingress traffic to their K8s resources directly from the deployment manifest, without configuring low-level network primitives — like DNS, IPs, NAT, and VPCs — outside of their K8s cluster. This makes it easy to add global traffic with security and scalability into K8s resources regardless of the underlying network infrastructure.
-
-For more details on the internal architecture, see [here](https://github.com/ngrok/kubernetes-ingress-controller/blob/main/docs/developer-guide/README.md).
+[Installation](#installation) | [Getting Started](https://ngrok.com/docs/using-ngrok-with/k8s/) | [Documentation](#documentation) | [Developer Guide](https://github.com/ngrok/kubernetes-ingress-controller/blob/main/docs/developer-guide/README.md) | [Known Issues](#known-issues)
 
 ## Installation
 
-> As of today, the ngrok Ingress Controller works only on ngrok accounts with the Pro subscription or above. If you would like to use the ngrok Ingress Controller with a free ngrok account, please [reach us out in our slack community](https://ngrok.com/slack) and we will be happy to help.
+### Helm
+
+> **Note** We recommend using the Helm chart to install the controller for a better experience through changes.
 
-The ngrok ingress controller is available as a helm chart. To add the ngrok helm chart repository, run:
+Add the ngrok Ingress Controller Helm chart:
 
-```bash
+```sh
 helm repo add ngrok https://ngrok.github.io/kubernetes-ingress-controller
 ```
 
-To install the latest version of the ngrok ingress controller, run the following command (setting the appropriate values for your environment):
+Then, install the latest version (setting the appropriate values for your environment):
 
-```bash
+```sh
 export NAMESPACE=[YOUR_K8S_NAMESPACE]
 export NGROK_AUTHTOKEN=[AUTHTOKEN]
 export NGROK_API_KEY=[API_KEY]
-helm install ngrok-ingress-controller ngrok/kubernetes-ingress-controller \
+
+helm install ngrok/kubernetes-ingress-controller \
+  --generate-name \
```

Comment:
I think this is a flag to emulate the old way of generating a name in helm2 and isn't very standard to see for most helm examples https://stackoverflow.com/questions/57961162/helm-install-unknown-flag-name

if this is an attempt resolve https://github.com/ngrok/kubernetes-ingress-controller/issues/179 we can go with for now and understand the best option in that issue

Reply:
Yes, as a part of the discussion that came out of #179 was that by using the name would append as the release name instead of when using generate-name it gets omitted and makes the names shorter. Would love to have that discussion with you at some time as well!

But, I agree, let's revert this and push it through #179 instead!